core/src/connection/pool.rs

```diff
@@ -183,8 +186,8 @@ where
         error: Option<ConnectionError<THandlerError<THandler>>>,
         /// A reference to the pool that used to manage the connection.
         pool: &'a mut Pool<THandler, TTrans>,
-        /// The remaining number of established connections to the same peer.
-        num_established: u32,
+        /// The remaining established connections to the same peer.
+        established_ids: Vec<ConnectionId>,
```

Comment:
```suggestion
        remaining_established_connection_ids: Vec<ConnectionId>,
```

Sorry, missed this one. For the sake of consistency with `NetworkEvent`.